examples/v0/reporting/get_hotel_ads_performance.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example gets Hotel-ads performance statistics for the 50 Hotel ad
```

Comment:
I know the Golden has "hotel-ads" hyphenated as well but I'm not sure why.

Reply:
@fiboknacky is there a reason for the hyphenation? It doesn't make sense grammatically, and the documentation doesn't seem to use it.

Reply:
I wasn't quite sure too that I was pointed out to use a hyphen or I did it myself.
Feel free to remove it.